stdlib/Profile/src/Profile.jl

```diff
@@ -31,6 +31,25 @@ macro profile(ex)
     end
 end
 
+export @profile_all
```

Comment:
I'd like to think a bit more about the right way to name this. Maybe we can bring this to a discussion in #profiling on julia slack, or tag someone on this PR for suggestions (like maybe `@IanButterworth` -- i didn't tag him yet so you guys can decide how to proceed).

My first instinct is that this doesn't seem clear enough to me. "All what," I would ask, upon reading this.
In go, this is called a "goroutine profile." Maybe we want to call it `@profile_tasks`?

Reply:
`@profile_tasks` seems good to me.

Will be changing to `@profile_tasks` for now, but there is a chance the API might change depending on the discussions upstream...